docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -1,42 +1,116 @@
+
+ifdef::downstream[]
+ifdef::context[:parent-context: {context}]
+:context: using-the-quarkus-extension-for-spring-di
+endif::[]
+
+include::attributes.adoc[]
+
+ifdef::downstream[]
+//file not present in upstream repo, but required for downstream content
+//commented out to allow build to pass
+//include::common/making-open-source-more-inclusive.adoc[]
+endif::[]
+
+ifdef::downstream[]
+[role="_abstract"]
+endif::[]
+As a developer, you can add the `quarkus-spring-di` extension to your project and inject dependencies in your {BrandName} application using Dependency Injection annotations provided by the Spring Framework.
+You can use Spring DI annotations as an alternative to MicroProfile CDI annotations.
+
+ifdef::upstream[]
+This guide is maintained in the main {BrandName} repository and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
-////
-= Quarkus - Quarkus Extension for Spring DI API
-
-include::./attributes.adoc[]
-:extension-status: preview
-
-While users are encouraged to use CDI annotations for injection, Quarkus provides a compatibility layer for Spring dependency injection in the form of the `spring-di` extension.
-
-This guide explains how a Quarkus application can leverage the well known Dependency Injection annotations included in the Spring Framework.
-
-include::./status-include.adoc[]
-
-== Prerequisites
-
-To complete this guide, you need:
-
-* less than 15 minutes
-* an IDE
-* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
-* Apache Maven {maven-version}
-
 
+//upstream version of notes about GitHub links
 == Solution
 
-We recommend that you follow the instructions in the next sections and create the application step by step.
-However, you can go right to the completed example.
+Follow the instructions in the next sections and create the application step by step.
+Alternatively, you can clone and use the completed example.
 
 Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
 
 The solution is located in the `spring-di-quickstart` {quickstarts-tree-url}/spring-di-quickstart[directory].
-
-== Creating the Maven project
-
-First, we need a new project. Create a new project with the following command:
-
+endif::[]
+
+//downstream version of notes about GitHub links with downstream-only attributes
+ifdef::downstream[]
+[NOTE]
+====
+You can follow this guide and create the example that uses the {PRODUCTNAME} extension for the Spring Web API, or you can download and view the completed example.
+To view the completed {ProductName} Spring Web example, download it as an link:{URL_GITHUB_QUICKSTARTS_REPO_ARCHIVE}[archive] or clone the {PRODUCTNAME} examples Git link:{URL_GITHUB_QUICKSTARTS_REPO_BASE}.git[repository].
+You can find the Spring Web example in the link:{URL_GITHUB_QUICKSTARTS_DIRECTORY_BASE}spring-di-quickstart[`spring-di-quickstart` directory].
+====
```

Comment:
IMO s/ProductName/BrandName - Downstream only attributes shouldn't be included in community. You can still use ProductName downstream as that is the convention. But I would avoid mixing downstream only stuff into community docs.

Reply:
Also you use `{BrandName}` in a downstream note here: https://github.com/quarkusio/quarkus/pull/14306/files#diff-2cfcbc66c9fd34b63ee3c4471ad5303b08037d6d344b534d17031538a748a5ccR77 

I think brandname is the way to go for consistency.